AdminRouter/config/adminrouter.conf

```diff
@@ -0,0 +1,576 @@
+#
+# This is the main Apache HTTP server configuration file.  It contains the
+# configuration directives that give the server its instructions.
+# See <URL:http://httpd.apache.org/docs/2.4/> for detailed information.
+# In particular, see 
+# <URL:http://httpd.apache.org/docs/2.4/mod/directives.html>
+# for a discussion of each configuration directive.
+#
+# Do NOT simply read the instructions in here without understanding
+# what they do.  They're here only as hints or reminders.  If you are unsure
+# consult the online docs. You have been warned.  
+#
+# Configuration and logfile names: If the filenames you specify for many
+# of the server's control files begin with "/" (or "drive:/" for Win32), the
+# server will use that explicit path.  If the filenames do *not* begin
+# with "/", the value of ServerRoot is prepended -- so "logs/access_log"
+# with ServerRoot set to "/usr/local/apache2" will be interpreted by the
+# server as "/usr/local/apache2/logs/access_log", whereas "/logs/access_log" 
+# will be interpreted as '/logs/access_log'.
+#
+# NOTE: Where filenames are specified, you must use forward slashes
+# instead of backslashes (e.g., "c:/apache" instead of "c:\apache").
+# If a drive letter is omitted, the drive on which httpd.exe is located
+# will be used by default.  It is recommended that you always supply
+# an explicit drive letter in absolute paths to avoid confusion.
+
+#
+# ServerRoot: The top of the directory tree under which the server's
+# configuration, error, and log files are kept.
+#
+# Do not add a slash at the end of the directory path.  If you point
+# ServerRoot at a non-local disk, be sure to specify a local disk on the
+# Mutex directive, if file-based mutexes are used.  If you wish to share the
+# same ServerRoot for multiple httpd daemons, you will need to change at
+# least PidFile.
+#
+ServerRoot "C:/DCOS/adminrouter/Apache24"
```

Comment:
I would add a parameter for the `adminrouter` directory instead of hard-coding the path here.

Something like:
```
ServerRoot "{{ adminrouter_install_dir }}/Apache24"
```

And during the setup script, you use the [Templating](https://github.com/dcos/dcos-windows/tree/master/scripts/Modules/Templating) PowerShell module to render the config file.

For example: This is [how](https://github.com/dcos/dcos-windows/blob/master/scripts/dcos-net-agent-setup.ps1#L131-L141) you would render a template like [this](https://github.com/dcos/dcos-windows/blob/master/scripts/templates/dcos-net/sys.config).

Reply:
addressed